src/touchWithMouseHOC.js

```diff
@@ -11,24 +11,34 @@ const mockEventTypes = {
   blur: 'touchcancel'
 }
 
+const distance = (x1, y1, x2, y2) =>
+  Math.sqrt(Math.pow(x2-x1, 2) + Math.pow(y2-y1, 2));
+
+const CLICK_TOLERANCE = 5;
```

Comment:
This feels magic. It's better we let the user decide the number.
Change the touchWithMouseHOC signature to `touchWithMouseHOC(Component, options)` or `touchWithMouseHOC(options)(Component)`